profet/pdb.py

```diff
@@ -79,12 +76,5 @@ def get_pdb(
                 pdb_id, PDBFileType(filetype), compression=True
             )
 
-        # Update the filename
-        filedir = filedir + "_" + pdb_id + "." + filetype
```

Comment:
this reverts back from the saving with pdbID when downloaded from pdb. Can we cache that too?

Reply:
Ah, sorry I didn't realise I had reverted something... I was obviously too hasty with my changes! I will address this now and update the pull request.